storage/pkg/config/defaultconfig.go

```diff
@@ -348,6 +348,10 @@ func DefaultConfig() *Config {
 				PublicEnableExpiredSharesCleanup: true,
 				PublicJanitorRunInterval:         60,
 				UserStorageMountID:               "",
+				Events: Events{
+					Address:   "127.0.0.1:4222",
```

Comment:
Sorry, this breaks because of my PR

Reply:
Thanks for pointing out, could you please check again?

Reply:
Oh already approved :+1: